src/styles/polygons/polygons_fragment.glsl

```diff
@@ -35,9 +40,14 @@ void main (void) {
     vec4 color = v_color;
     vec3 normal = TANGRAM_NORMAL;
 
-    // Normal material texture (fragment lighting only)
-    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
-        calculateNormal(normal);
```

Comment:
Confirmed with a local build!

On Thu, Apr 14, 2016 at 9:06 PM, Brett Camper <notifications@github.com>
wrote:

> In src/styles/polygons/polygons_fragment.glsl
> <https://github.com/tangrams/tangram/pull/282#discussion_r59815651>:
>
> > @@ -35,9 +40,14 @@ void main (void) {
> >      vec4 color = v_color;
> >      vec3 normal = TANGRAM_NORMAL;
> >
> > -    // Normal material texture (fragment lighting only)
> > -    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
> > -        calculateNormal(normal);
>
> @karimnaaji <https://github.com/karimnaaji> I followed your suggestion
> and this should be fixed here (verified in Tangram Play): 9113f3a
> <https://github.com/tangrams/tangram/commit/9113f3aab36a63af7ea9f6b6a18efc1424a17b0d>
>
> Maybe someone can double-check this and then we can cut a new release.
> Thanks for finding!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram/pull/282/files/0bcb36919e2e553c47a179fe5c07bcd9e27e4491#r59815651>
>